tests/phpunit/integration/MediaWiki/Api/EditEndpointTest.php

```diff
@@ -302,4 +304,10 @@ private function countItemsInDatabase(): int {
 		);
 	}
 
+	private function getMockSessionProvider( bool $safeAgainstCSRF ): SessionProviderInterface {
```

Comment:
Hm, there’s no test that passes `false` here?

Reply:
Added a test, I hope it looks okay.

Reply:
nice :+1: